docs/applications/github-action/01-aws.md

```diff
@@ -1,8 +1,8 @@
 ---
-id: applications-container-repositories-aws
-slug: /applications/container-repositories/aws
-title: AWS Container Repositories
-sidebar_label: AWS Container Repositories
+id: applications-github-action-aws
+slug: /applications/github-action/aws
```

Comment:
These are just so the pages don't break until I can update the docs for these clouds specifically.